frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```
@@ -161,6 +152,101 @@ export const FormComponentConfig = ({
+          key: propertyKey,
+          enumValues: properties[propertyKey]?.enum,
+        };
+        return isPreselectedOptionIndex ? (
+          <CollapsiblePropertyEditor
+            key={propertyKey}
+            label={t('ux_editor.component_properties.preselectedOptionIndex_button')}
+          >
+            <EditNumberValue {...commonProps} />
+          </CollapsiblePropertyEditor>
+        ) : (
+          <EditNumberValue {...commonProps} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)